hack/lib/golang.sh

```diff
@@ -349,6 +358,20 @@ kube::golang::build_binaries_for_platform() {
     fi
   fi
 
+  for test in "${tests[@]:+${tests[@]}}"; do
+    local outfile=$(kube::golang::output_filename_for_binary "${test}" \
+      "${platform}")
+    # Go 1.4 added -o to control where the binary is saved, but Go 1.3 doesn't
+    # have this flag. Whenever we deprecate go 1.3, update to use -o instead of
+    # changing into the output directory.
+    local orig_path=$(pwd)
```

Comment:
nit: Neater to use pushd and popd, which does this for you.  Seems to work on linux, mac os x, windows.  But I guess that introduces an extra dependency. Up to you.